.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,105 @@
+name: Bug Report Form
+description: Report an issue related to the Home Assistant Operating System.
+labels: bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Make sure to test with the last version of the Operating System
+        before reporting a bug. If the bug appears to be a regression,
+        make sure to check if the bug indeed disappears from the
+        previous version. Use `ha os update --version x.y` command to
```

Comment:
Maybe link to documentation for this? <https://www.home-assistant.io/common-tasks/os/#run-a-specific-version>

Reply:
Yeah I was thinking about it, but it documents how to run a specific Core version. We should add a section how to run a specific OS version there too.

Reply:
Og right! my bad.
maybe we should add it to the docs? :smile: